content/en/docs/architecture/timeouts.md

```diff
@@ -227,6 +227,10 @@ for (( i = 0; i < 999; i++ )); do
 done
 ```
 
+{{< alert title="Warning" color="warning" >}}
+You have to be careful with the `trap` specification. If you specify `trap cleanup TERM EXIT`, the `cleanup` function will be called twice: once when the `SIGTERM` is received, and once when the shell exits. This is not what you want. Instead, specify `trap cleanup TERM` to only call the `cleanup` function when the `SIGTERM` is received. If you specify `trap cleanup EXIT`, the `cleanup` function will catch the `SIGTERM`, but will ignore the `exit `status (in this case `1`). The prefix `SIG-` is not suggested, as it is not portable to all shells.
```

Comment:
This is a bit too strong.  For regular steps which want to perform the same action on interruption and termination, `EXIT` is reasonable.  I'd just describe what the effect of each combination is and let people decide how the want to write their handler(s): `TERM` is only called on receipt of the signal, `EXIT` is always called on signal and regular exit and ignores the `exit` status from its function, etc.

Also make it a list (`<ul>`) so it's a bit easier to parse.